operator/builtin/input/file/file_test.go

```diff
@@ -205,6 +205,71 @@ func TestReadExistingLogs(t *testing.T) {
 	waitForMessage(t, logReceived, "testlog2")
 }
 
+// TestReadUsingNopEncoding tests when nop encoding is set, that the splitfunction returns all bytes unchanged.
+func TestReadUsingNopEncoding(t *testing.T) {
+	tcs := []struct {
+		testName string
+		input    []byte
+		test     func(*testing.T, chan *entry.Entry)
+	}{
+		{
+			"simple",
+			[]byte("testlog1"),
+			func(t *testing.T, c chan *entry.Entry) {
+				waitForMessage(t, c, "testlog1")
```

Comment:
A really important part of this feature is that the resulting log body should be a `[]byte`.

Reply:
Updated to dump bytes on the emit func. Instead of decoding it, it just drops the bytes into the entry body.